src/components/views/rooms/MessageComposer.js

```diff
@@ -48,9 +49,10 @@ export default class MessageComposer extends React.Component {
             inputState: {
                 style: [],
                 blockType: null,
-                isRichtextEnabled: true,
+                isRichtextEnabled: UserSettingsStore.getSyncedSetting('MessageComposerInput.isRichTextEnabled', true),
+                wordCount: 0,
             },
-            showFormatting: false,
+            showFormatting: UserSettingsStore.getSyncedSetting('MessageComposer.showFormatting', false),
```

Comment:
What, whether it defaults to false or whether it persists correctly? It does seem to persist correctly.